packages/venia-concept/src/components/MiniCart/section.css

```diff
@@ -0,0 +1,19 @@
+li > button {
```

Comment:
I'm sure we don't want to globally style all `button` elements that happen to be descendants of `li` elements. 😉 

- Avoid tagname selectors wherever possible
- The first selector within a complex selector (such as this) should always be a classname
- Only one descendant combinator per selector